packages/terra-modal-manager/tests/wdio/modal-manager-spec.js

```diff
@@ -345,4 +325,18 @@ Terra.describeViewports('ModalManager - Behaviors', ['large'], () => {
       });
     });
   });
+
+  describe('Component Integration', () => {
```

Comment:
Moved this to be its own separate describe block outside of the focus management describe block.